package/com.unity.formats.usd/Dependencies/USD.NET.Unity/Geometry/PointInstancerSample.cs

```diff
@@ -48,16 +48,18 @@ public Matrix4x4[] ComputeInstanceMatrices(Scene scene, string primPath)
             var prim = scene.GetPrimAtPath(primPath);
             var pi = new pxr.UsdGeomPointInstancer(prim);
             var xforms = new pxr.VtMatrix4dArray();
+            var timeCode = scene.Time == null ? pxr.UsdTimeCode.Default() : scene.Time;
 
             // For the base time from Pixar docs:
             // "If your application does not care about off-sample interpolation, it can supply the same value for baseTime that it does for time."
-            pi.ComputeInstanceTransformsAtTime(xforms, scene.Time == null ? pxr.UsdTimeCode.Default() : scene.Time, scene.Time == null ? pxr.UsdTimeCode.Default() : scene.Time);
```

Comment:
nice refactor :)

Reply:
Thanks! but its actually part of Jeremy's [fix](https://jira.unity3d.com/browse/USDU-305) 😆